### PR TITLE
bView disconnects randomly #471

### LINKED ISSUE #471: bView disconnects randomly
This happened again (V1.9.4). The workaround doesn't seem to work. It works if I close the bView, but doesn't work for the 'disconnect' scenario

### PR DESCRIPTION
**Pull request by rd1992**
Going for a reset if the browser stops responding for whatever reason.
Fixes #471

### CODE REVIEW

File: src/browserview/BrowserComponent.java [hunk 1]
```diff
@@ -252,9 +252,11 @@ public void jumpToComment(){
 	
 	private boolean isBrowserActive(){
 		if (driver.getCurrentUrl().isEmpty() && driver != null){
+			logger.warn("Unable to read url from bview. Resetting.");
 			return false;
 		}
 		else if (driver == null){
+			logger.warn("chromedriver process was killed !");
 			return false;
 		}
```
Comment (dariusf) [inline src/browserview/BrowserComponent.java]: The logic here could be structured like this:

```java
if (driver == null) {
    // chromedriver process killed
}
// driver != null from here on
else if (driver.getCurrentUrl() == null /* extra null check */ || driver.getCurrentUrl().isEmpty()) {
    // unable to read url from browser
}
```

Also, is it possible that `getCurrentUrl()` may throw `WebDriverException`? I looked at the docs and there was no mention of it, but then `WebDriverException` is unchecked, so...

**Comment by damithc** [comment]
Can we wrap this up before today's dev release?

File: src/browserview/BrowserComponent.java (hunk 1 above)
Comment (rd1992) [inline src/browserview/BrowserComponent.java]: hmm it is possible that it throws this exception. shall impement a try catch block here

**Comment by rd1992** [comment]
I have taken @dariusf 's suggestions and pushed the changes.

**Pull request merged**